integrated_channels/moodle/models.py

```diff
@@ -64,6 +66,38 @@ class MoodleEnterpriseCustomerConfiguration(EnterpriseCustomerPluginConfiguratio
         )
     )
 
+    decrypted_username = EncryptedCharField(
+        max_length=255,
+        verbose_name="Encrypted Webservice Username",
+        blank=True,
+        help_text=_(
+            "The encrypted API user's username used to obtain new tokens."),
+        default="",
+    )
```

Comment:
[nit] Once we address the above feedback, please make sure to add back null=True and remove default value.

Reply:
Why did we use this filed name decrypted_username except for encrypted_username?

Reply:
@sameenfatima78 @zamanafzal "decrypted" is used here to remain consistent with other usage of fernet fields in code, this decrypted fields return unencrypted values when referenced in code.